### DISCUSSION TITLE
Working together

### DISCUSSION THREAD

**Discussion by Reagan27** [Chunnyluny-RimWorld-Modding/DogApparel#108] [Share gamescreenshot funnies with the mod visible!]
I will like to work with you together on c# projects

**Comment by Chunnyluny** [comment]
that sounds like a plan, do you have discord or any experience with C# and rimworld mods ? there will be a 1.5 release which has some changes in pawn rendering, I hope you will understand

**Reply by Reagan27** [reply]
Hello, yes, I do. This is my email 
luttareagan22@gmail.com . 
We can talk more about it. When do we start the project?

**Reply by Chunnyluny** [reply]
I don't have enough storage left on my Gmail account. Btw, I already started with Dogapparel mod. But I'm not sure how to implement the ideas that I already have into c#. The ideas are in the README of the repository. It would also be nice if you had discord, so I have things more organized.

**Reply by Chunnyluny** [reply]
I got a new email address for coding and modding, you can contact me here
chunnylunycoding.modding@gmail.com and sent you a message through Google chat.
